sdk/spring/azure-spring-data-cosmos/CHANGELOG.md

```diff
@@ -1,16 +1,26 @@
 ## Release History
 
-### 3.45.0-beta.1 (Unreleased)
+### 3.46.0-beta.1 (Unreleased)
```

Comment:
Please add a change log entry for this PR.

Reply:
I see it below but it is referencing the wrong PR (40022) where this is PR 40096, this also should be up in the 3.46 section instead of 3.45 I believe.